exo/networking/manual/test_manual_discovery.py

```diff
@@ -0,0 +1,103 @@
+import asyncio
```

Comment:
Not sure what sort of structure you're trying to follow for unittests, so I just tried to follow suit with UDP & tailscale's tests. LMK if there are other things you think I should test or include

Reply:
all good.